book/graphics.md

```diff
@@ -591,6 +591,13 @@ window's new width and height can be found in the `width` and `height`
 fields on the event object. Remember that when the window is resized,
 the line breaks must change, so you will need to call `layout` again.
 
+*about:blank:* Currently, a malformed URL causes the browser to crash.
```

Comment:
Any reason this goes in Chapter 2 instead of 1? Just because we need more in Chapter 2?

Reply:
Yes: chapter 1 doesn't have a browser, it's just a commandline tool. It makes more sense to me to justify this when there is a browser UI to think about.